exif2timestream.py

```diff
@@ -581,6 +689,11 @@ def main(opts):
     # beginneth the actual main loop
     start_time = time()
     cameras = parse_camera_config_csv(opts["-c"])
+    try:
+        global EXIF_DATE_MASK
+        EXIF_DATE_MASK = opts["-m"]
```

Comment:
Ah, that's not how `docopt` works, if I understand what you want to do correctly. `'-m'` will always be in the dict, however if it isn't specified, it will be `None`. So use 
```python
if opts['-m'] is not None:
    EXIF_DATE_MASK = opts['-m']
```